.gitignore

```diff
@@ -1,6 +1,7 @@
 .gradle
 /build/
 !gradle/wrapper/gradle-wrapper.jar
+/src/main/resources/application.properties
```

Comment:
w pracy takich rzeczy nie robimy :) po co to dodałeś do gitignore?

Reply:
Bo dostałem ostrzeżenie, że ktoś się logował na tą pocztę. Niby jest tylko dla testów, ale postanowiłem zmienić i ukryć hasło.